### PR TITLE
Initial getting codes lined up pull request

### PR DESCRIPTION
**Pull request by FlanFlanagan**

### CODE REVIEW

**Comment by scopatz** [comment]
Is this ready for another review round @FlanFlanagan?

**Comment by FlanFlanagan** [comment]
Yes, please

File: xsgen/brightlite.py [hunk 1]
```diff
@@ -27,14 +30,30 @@ def write(self, libs, dirname):
         for mat, matlib in libs.items():
             if isinstance(mat, int):
                 fname = str(nucname.zzaaam(mat))
+            #elif mat == 'fuel':
+            #    fname = mat
             else:
-                fname = mat
+                continue
```
Comment (scopatz) [inline xsgen/brightlite.py]: please remove the commented lines.

File: xsgen/brightlite.py [hunk 2]
```diff
@@ -27,14 +30,30 @@ def write(self, libs, dirname):
...
+                    try:
+                        trans_matrix[nuc_name].append(matlib['material'][i].comp[temp_nuc])
+                    except KeyError:
+                        if matlib['material'][i].comp[temp_nuc] > 1.E-4:
+                            zero_array = [0]*i
+                            trans_matrix[nuc_name] = zero_array
+                            trans_matrix[nuc_name].append(matlib['material'][i].comp[temp_nuc])
+                i+=1
             nucs = matlib["tracked_nucs"]
+            #print(trans_matrix)
```
Comment (scopatz) [inline xsgen/brightlite.py]: remove

File: xsgen/brightlite.py [hunk 3]
```diff
@@ -27,14 +30,30 @@ def write(self, libs, dirname):
...
                      for row in rownames]
+            trans_matrix = {}
+            i = 0
+            while i < len(matlib['material']):
+                for temp_nuc in matlib['material'][i].comp:
+                    nuc_name = str(nucname.name(temp_nuc))
+                    try:
+                        trans_matrix[nuc_name].append(matlib['material'][i].comp[temp_nuc])
+                    except KeyError:
+                        if matlib['material'][i].comp[temp_nuc] > 1.E-4:
```
Comment (scopatz) [inline xsgen/brightlite.py]: There shouldn't be any magic threshold numbers. These should be user-configurable.

File: xsgen/brightlite.py [hunk 4]
```diff
@@ -65,3 +84,14 @@ def write(self, libs, dirname):
             f.write("\n".join(cladrows))
             f.write("\n")
         shutil.copyfile("TAPE9.INP", os.path.join(dirname, "TAPE9.INP"))
+        # write cross section json file - this section is now causing a runtime error
```
Comment (scopatz) [inline xsgen/brightlite.py]: Can you provide more details please?

File: xsgen/openmc_origen.py [hunk 5]
```diff
@@ -151,15 +153,16 @@ def __init__(self, rc):
...
-                        src_group_struct=np.logspace(1, -9, 1001))
+                        src_group_struct=rc.openmc_group_struct)
         data_sources = [self.omcds]
         if not rc.is_thermal:
             data_sources.append(self.eafds)
-        data_sources += [data_source.SimpleDataSource(),
-                         data_source.NullDataSource()]
-        for ds in data_sources[1:]:
+        #data_sources += [data_source.SimpleDataSource(),
+        #                 data_source.NullDataSource()]
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: Maybe keeping the SimpleDataSource is a good idea? That or just remove them both completely

File: xsgen/openmc_origen.py [hunk 6]
```diff
@@ -151,15 +153,16 @@ def __init__(self, rc):
...
+        for ds in data_sources[0:]:
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: `data_sources[0:]` is the same as `data_sources[:]`, which in a loop is equivalent to `data_sources`

File: xsgen/openmc_origen.py [hunk 7]
```diff
@@ -491,7 +528,6 @@ def _make_omc_input(self, state):
         ctx['_eafds_egrid'] = " ".join(map(str, sorted(self.eafds.src_group_struct)))
         ctx['_omcds_egrid'] = " ".join(map(str, sorted(self.omcds.src_group_struct)))
         # nucs = core_nucs & valid_nucs
-        # ctx['_nucs'] = " ".join([nucname.serpent(nuc) for nuc in sorted(nucs)])
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: remove these lines?

File: xsgen/openmc_origen.py [hunk 8]
```diff
@@ -599,6 +644,9 @@ def _make_origen_input(self, transmute_time, phi_tot, mat):
         """
         # may need to filter tape4 for Bad Nuclides
         # if sum(mat.comp.values()) > 1:
+        for nuc in mat.comp:
+            if mat.comp[nuc] < 1e-5:
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: Threshold should not be hardcoded.

File: xsgen/openmc_origen.py [hunk 9]
```diff
@@ -715,4 +762,8 @@ def _origen(origen_params):
         "material": list(out_mat.comp.items()),
         "phi_tot": phi_tot
         })
+    #if burnup < 0.0:
+    #    msg = 'Negative burnup found for {0}:\n{1}'
+    #    msg = msg.format(mat_id, pformat(results[1]))
+    #    raise ValueError(msg)
```
Comment (scopatz) [inline xsgen/openmc_origen.py]: This seems valid.

**Pull request merged**